wake-lock/wakelock-disabled-by-feature-policy.https.sub.html

```diff
@@ -11,13 +11,13 @@
   const cross_origin_src =
     "https://{{domains[www]}}:{{ports[https][0]}}" + same_origin_src;
 
-  promise_test(t => {
-    return promise_rejects(t, "SecurityError", navigator.getWakeLock("screen"));
+  test(() => {
+    assert_throws("SecurityError", () => new WakeLock("screen"));
```

Comment:
We moved to NotAllowedError as that is more correct

Reply:
Oh, I may miss the change. Will fix soon.

Reply:
Done. :)